server/gitrest/packages/gitrest-base/src/routes/summaries.ts

```diff
@@ -93,10 +103,10 @@ async function createSummary(
             writeSummaryResponse.id,
         ).catch((err) => {
```

Comment:
>err [](http://example.com/codeflow?start=17&length=3)

(nit) use error to be consistent?

Reply:
Fixed it (and some other occurrences in this file). Some other places in GitRest still use a different pattern (sometimes `err`, sometimes `e`) and that is also the case for the whole repo. I will not change those here to avoid making this PR much larger

Reply:
Makes sense :) Thanks for the fix!